tags/tags-impl/impl/src/java/org/sakaiproject/tags/impl/TagServiceImpl.java

```diff
@@ -138,12 +138,15 @@ public void updateTagAssociations(String collectionId, String itemId, List<Strin
                 continue;
             }
             // new association, check tag exists
```

Comment:
Could we move the comment below the added code?

Reply:
Yes! :+1:

Reply:
Thanks!